.github/workflows/submit.yml

```diff
@@ -1,4 +1,4 @@
-name: "Submit to Web Store"
+name: 'Submit to Web Store'
```

Comment:
Do we need to make this change?

Reply:
No, I could exclude this file from Prettier

Reply:
I trust your discretion on that.